srfi-tools/generate.sld

```diff
@@ -1,23 +1,32 @@
 (define-library (srfi-tools generate)
   (export write-single-srfi-landing-page
 	  write-single-srfi-readme
+          write-single-srfi-document
 	  write-srfi-home-page
 	  write-srfi-landing-pages
 	  write-srfi-list-subscribe-page
 	  write-srfi-readmes)
   (import (scheme base)
-
+          (scheme file)
+          (scheme read)
+          (scheme write)
+          (srfi 1)
```

Comment:
Please import from `(srfi-tools private list)`. We use `private` shims to import from all SRFIs, since some code is vendored or `cond-expand`ed for wider portability.